release_creation/bundle/requirements/v1.8.pre.requirements.txt

```diff
@@ -0,0 +1,18 @@
+hatchling<2.0
```

Comment:
Can you please leave a remark about how this list is generated?

It would be useful to see what's changed from last time.

Reply:
It's mostly copy/paste of the prior version and replace 1.7 with 1.8. `hatchling` is new though. Traditionally this would get documented via git diffs, but it's different files. I wonder if it might be worth committing the 1.7 reqs file as the new 1.8 file against main, then creating this PR. That way we could use git blame to do it. Thoughts?

Reply:
@aranke I created this [PR](https://github.com/dbt-labs/dbt-core-bundles/pull/130), which should be merged ahead of these changes. This would allow us to comment on the particular changes here.